File: q5.c
//SMALLEST MULTIPLE
#include<stdio.h>
void main()
{
     int a=1,k=1;
     while(a!=0)
     {
          int flag=0;
          for(int i=1;i<=20;i++)
          {
               if(k%i!=0)
                    break;
                    else if(i==20)
                    {
                         flag=1;
                         break;
                    }
          }
          k++;
          if (flag==1)
          {
               a=0;
               printf("\n%d is the smallest positive no. that is evenly divisible by all of the numbers from 1 t0 20.",k-1);
          }
     }
}

File: q5.java
package hello;

public class q5
{

	public static void main(String[] args) 
	{
		long tlcm=1l;
		for(long i=1l;i<=20;i++)
			tlcm=lcmf(i,tlcm);
		System.out.println(tlcm+" is the smallest positive no. that is evenly divisibe by all of the numbers from 1 to 20 .");
	}
	static long lcmf(long a,long b)
	{
		long m=a,n=b,gcd=0;
		while(a!=b)
		{
			if(a>b)
				a-=b;
			else
				b-=a;
		}
		gcd=a;
		return (m*n)/gcd;
			
		
	}

}
